layouts/partials/posts/series.html

```diff
@@ -0,0 +1,23 @@
+{{ $currentPageUrl := .URL }}
+{{ if .Params.series }}
+<section class="see-also">
+  {{ range .Params.series }}
+    <h3>See also in {{ . }}</h3>
+    {{ $name := . | urlize }}
+    {{ $series := index $.Site.Taxonomies.series $name }}
+    {{ if gt (len $series.Pages) 0 }}
+      <nav class="list">
```

Comment:
```suggestion
      <nav>
```
No need to use this class here.